velox/exec/HashBuild.h

```diff
@@ -42,17 +42,20 @@ class HashBuild final : public Operator {
   enum class State {
     /// The running state.
     kRunning = 1,
+    /// The yield state that voluntarily yield cpu after running too long when
+    /// processing input from spilled file.
+    kYield = 2,
```

Comment:
Curious, why do we need to assign an integer value to each enum?

Reply:
Hmm, yeah, it is not necessary but I always explicitly set value to enum in practice.